### ISSUE TITLE
def feature_transform_reguliarzer(trans):

### ISSUE THREAD

**Issue by jeannotes** [fxia22/pointnet.pytorch#40]
Hi, your loss implemention is like this ||(A*(A_T-I))||, but the loss should be like 
||(I-A*A_T)||, therefore, it should be like this    
loss = torch.mean(torch.norm(torch.bmm(trans, trans.transpose(2, 1) ) - I, dim=(1, 2))),
am I right?

**Comment by fxia22** [comment]
You are right, it should be `loss = torch.mean(torch.norm(torch.bmm(trans, trans.transpose(2,1)) - I, dim=(1,2)))`

**Comment by fxia22** [comment]
I also updated the benchmarking results.
